classes/springbot_footer.php

```diff
@@ -39,17 +39,21 @@ public function show_async_script() {
 					echo "  pixel.className = 'sb-pixel';\n";
 					echo "  pixelContainer.appendChild(pixel);\n";
 					echo "  document.body.appendChild(pixelContainer);\n";
-
-					// Set the product_id for our async script to use if needed
-					echo "  var Springbot = Springbot || {};\n";
-					echo "  Springbot.product_id = \"{$product->get_id()}\";\n";
 					echo "});\n";
 
 				}
 
 				echo "  })();\n";
 				echo "</script>\n";
 
+				if ( is_product() && ( $product instanceof WC_Product ) ) {
```

Comment:
This would be cleaner if we still kept all this code inside that 1 script tag, and just move this part out of the load event's callback!

Reply:
I had to do it this way because everything else is loaded in a function, and that also blocked it from loading. So I moved it out of the other code and made it stand alone.